cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaExporter.java

```diff
@@ -266,4 +266,26 @@ public Secret generateSecret(ClusterCa clusterCa, boolean isMaintenanceTimeWindo
         return ModelUtils.buildSecret(reconciliation, clusterCa, secret, namespace, KafkaExporterResources.secretName(cluster), componentName,
                 "kafka-exporter", labels, ownerReference, isMaintenanceTimeWindowsSatisfied);
     }
+
+    /**
+     * Generates the NetworkPolicies relevant for Kafka Exporter
+     *
+     * @return The network policy.
+     */
+    public NetworkPolicy generateNetworkPolicy() {
+        // List of network policy rules for all ports
+        List<NetworkPolicyIngressRule> rules = new ArrayList<>();
+
+        // Everyone can access metrics
+        rules.add(NetworkPolicyUtils.createIngressRule(MetricsModel.METRICS_PORT, List.of()));
+
+        // Build the final network policy with all rules covering all the ports
+        return NetworkPolicyUtils.createNetworkPolicy(
+                KafkaExporterResources.networkPolicyName(reconciliation.name()),
```

Comment:
I would just use `componentName` here.

```suggestion
                componentName,
```